ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -325,6 +325,30 @@
   @Default("false")
   public final boolean storeSetFilePermissionEnabled;
 
+  /**
+   * Whether to enable auto close last log segment during compaction.
+   */
+  @Config(storeAutoCloseLastLogSegmentEnabledName)
+  @Default("false")
+  public final boolean storeAutoCloseLastLogSegmentEnabled;
+  public static final String storeAutoCloseLastLogSegmentEnabledName = "store.auto.close.last.log.segment.enabled";
+
+  /**
+   * Specifies the minimum value for maxLagForPartition which can unseal replica.
+   */
+  @Config(storeUnsealReplicaMinimumLagName)
+  @Default("10485760")
```

Comment:
Actually after a second thought, I think maybe it's better to set to 0. Since I checked the metric and most of time the lag is zero after it fully catch up with it's peer replica after token gets reset.